src/concept-coach/base.cjsx

```diff
@@ -18,6 +19,8 @@ User = require '../user/model'
 
 {channel} = require './model'
 
+VIEWS = ['loading', 'login', 'registration', ['task', 'progress', 'profile', 'dashboard', 'registration']]
```

Comment:
It's not clear to me (except when reading this diff) that the array of views correspond to the `authLevel` determined below.